src/utils/uint8arr.ts

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
This file is not actually needed. Use corresponding functions from `ethereumjs-utils`

Reply:
[a79cec9](https://github.com/Consensys/sol-dbg/pull/41/commits/a79cec9eee0cdf560102b9d5cd98f305257c2250)